src/js/components/Anchor/index.d.ts

```diff
@@ -5,6 +5,7 @@ export interface AnchorProps {
   a11yTitle?: string;
   alignSelf?: AlignSelfType;
   gridArea?: string;
+  disabled?: boolean;
```

Comment:
I think TS should get it from the HTML, do we really need it?

Reply:
There is a disabledStyle in the styledAnchor.js I do not think that disabled is getting it from HTML because its a style that is in grommet

Reply:
https://www.w3schools.com/tags/att_disabled.asp the disabled attribute is not for an a tag that is being used in the StyledAnchor

Reply:
yeap +1.  no disabled in the <a> html so i think we need it.